# Publish debug Topics
publish_debug_topics: true

# Link names
robot_link_name: base_link #Robot link name published by Gazebo plugin
station_link_name: station_base_link #Charging-station link name published by Gazebo plugin

# Docking parameters
robot_connector_offset: -0.11  #Distance from robot origin to the charging connector in x-axis direction [m]
station_connector_offset: 0.121 #Distance from charging-station origin to the charging connector in the x-axis direction [m]
docking_area_radious: 0.1 #Radius of the area which publish the charging voltage [m] 0.15
docking_area_angle: 0.5 #Range of angles to publish charging voltage (+- "publih_voltage_area_angle" [m])
voltage: 36.0 #Charging voltage whole docking